packages/ember-metal/lib/property_set.js

```diff
@@ -98,10 +97,9 @@ function setPath(root, path, value, tolerant) {
   assert('Property set failed: You passed an empty path', keyName.trim().length > 0)
 
   let newPath = parts.join('.');
-
   let newRoot = getPath(root, newPath);
 
-  if (newRoot) {
+  if (newRoot !== null && newRoot !== undefined && !newRoot.isDestroyed) {
```

Comment:
We shouldn't be hard coding `.isDestroyed` check here.  We moved the flag into meta, we should use that...

Reply:
this is already handled in https://github.com/emberjs/ember.js/blob/master/packages/ember-metal/lib/property_set.ts#L68